artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/mirror/AMQPMirrorControllerSource.java

```diff
@@ -338,6 +367,12 @@ Message copyMessageForPaging(Message message) {
    public void sendMessage(Transaction tx, Message message, RoutingContext context) {
       SimpleString address = context.getAddress(message);
 
+      if (isBlockedByNoForward(message)) {
+         String remoteID = getRemoteMirrorId();
```

Comment:
Variable looks unused, did you mean to add it to the logging?

Reply:
It's a leftover from a debugging session I forgot to remove.